gui/src/components/Suites.js

```diff
@@ -25,15 +26,9 @@ function Suites({socket}) {
       default:
         break;
     }
-  };
+  }, [suites, setSuite]);
   
-  socket.onmessage = (event) => {
-    const data = event.data;
-    const jsonData = JSON.parse(data);
-    console.log(`Recived message ${data}`);
-    addAndUpdateSuites(suites, jsonData);
-    console.log(suites);
-  };
+  useEffect(() => addAndUpdateSuites(suiteMessage), [suiteMessage]);
```

Comment:
Getting an error from this line
`front_end | [0] Line 31:53: React Hook useEffect has a missing dependency: 'addAndUpdateSuites'. Either include it or remove the dependency array react-hooks/exhaustive-deps`